cms/templates/js/content-visibility-editor.underscore

```diff
@@ -0,0 +1,33 @@
+<form>
+<h3 class="modal-section-title" id="content_visibility_label"><%- gettext('Student Visibility') %></h3>
+<div class="modal-section-content staff-lock">
+    <ul class="list-fields list-input content-visibility" role="group" aria-labelledby="content_visibility_label">
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="staff_only">
+                <%- gettext('Hide from students') %>
+            </label>
+        </li>
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="hide_after_due">
+                <%- gettext('Hide content but not grades from students after due date') %>
+            </label>
+        </li>
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="visible">
+                <%- gettext('Always show to students (Default)') %>
+            </label>
+        </li>
+    </ul>
+
+    <% if (hasExplicitStaffLock && !ancestorLocked) { %>
+        <p class="tip tip-warning">
+            <% var message = gettext('If you make this %(xblockType)s visible to students, students will be able to see its content after the release date has passed and you have published the unit.'); %>
```

Comment:
This is a context sensitive tip.

If:
- the subsection is explicitly set to `staff_only` and
- there is no inherited lock from the containing section

Then:
- selecting an option that would make the content visible to students (either of the not `staff_only` choices)
makes the tip visible.

I think it's intended to be a warning for the situation of unhiding content. The same pattern is used here for the 2-state staff lock checkbox: https://github.com/edx/edx-platform/blob/efischer/hidden_studio/cms/templates/js/staff-lock-editor.underscore#L13-L28